python/src/equistore/operations/to.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import torch
+
+from ..block import TensorBlock
+from ..labels import Labels
+from ..tensor import TensorMap
+
+
+def to(
```

Comment:
I would rather have two separate functions for TensorMap and TensorBlock, like the rest of the operations API